actix-session/src/storage/redis_rs.rs

```diff
@@ -2,7 +2,13 @@ use std::sync::Arc;
 
 use actix_web::cookie::time::Duration;
 use anyhow::Error;
-use redis::{aio::ConnectionManager, AsyncCommands, Cmd, FromRedisValue, RedisResult, Value};
+#[cfg(not(feature = "redis-rs-session"))]
+use deadpool_redis::{
```

Comment:
Add
```rust
use deadpool_redis::redis as redis;
```
?

Reply:
Thanks, I didn't even think of that. I'm already applying all that to avoid certain _shadowing_ and thus also reinforce better reading.